src/services/TestDataService.ts

```diff
@@ -1111,6 +1112,22 @@ export class TestDataService {
         return salesChannel;
     }
 
+    async addSalesChannelLanguage(salesChannelId: string, languageCode: string) {
```

Comment:
I would change to `assignSalesChannelLanguage` to match the ones above

Reply:
I'm using add trying to describe that I'm adding another language to the sales channel, which already has a default language.

Reply:
I would also recommend to stick to our language slang with `assign` instead of `add`